dom/abort/event.any.js

```diff
@@ -0,0 +1,22 @@
+async_test(t => {
+  const c = new AbortController(),
+        s = c.signal;
+  let state = "begin";
+
+  assert_false(s.aborted);
+
+  s.addEventListener("abort",
+    t.step_func_done(e => {
```

Comment:
This will done() the test twice, once via the step_func_done and once via t.done() below